src/client/all_meals.js

```diff
@@ -0,0 +1,82 @@
+import React, { useState, useEffect, useCallback } from "react";
+
+import ShowUser from "./showmeals";
+
+const AllMeals = () => {
+  const [name, setName] = useState("");
+
+  const [users, setUsers] = useState([]);
+
+  const [isLoading, setIsLoading] = useState(true);
+  
+  const [isEmpty, setEmpty] = useState(true);
+
+  const apiBaseUrl = "http://localhost:3000/api/meals";
+
+  const SearchUser = useCallback((value) => {
+    setUsers([]);
+    setEmpty(true);
+
+    console.log("in search user");
+    let apiUrl = "http://localhost:3000/api/meals";
```

Comment:
we have to fetch from /api/meals since we deployed so we will not use the local data

Reply:
done!thanks for the feedback!